appyters/scRNA_seq/Single_cell_RNA_seq_Analysis_Pipeline.ipynb

```diff
@@ -1029,11 +1030,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%%appyter code_exec\n",
     "# basic qc\n",
```

Comment:
This comment being here will result in an empty cell with just
```
# basic qc
```

Maybe you meant to put the comment in the if statement so that the cell is hidden otherwise?